data/src/main/java/com/bottlerocketstudios/brarchitecture/data/model/ResponseToApiResultMapper.kt

```diff
@@ -38,12 +41,22 @@ class ResponseToApiResultMapperImpl : ResponseToApiResultMapper {
         return when {
             response.isSuccessful -> Status.Success(Unit)
             else -> {
-                Timber.w("[toEmptyResult] Api not successful: message ${response.message()} code: ${response.code()}")
+                Timber.w("[toCodedResult] Api not successful: message ${response.message()} code: ${response.code()}")
```

Comment:
@MMurrayAtBR Revert this change so that the logs use `toEmptyResult` (which matches the function name :)